### ISSUE TITLE
(Fixed in KEDR 0.4) KEDR should detect loading of module with '-' in its name

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/kedr#2]
```
Inside Linux kernel all '-' characters in module name are internally replaced 
with '_'.
KEDR should take this into account when detect target module loading.

E.g., when running with

# kedr start ideapad-laptop

KEDR should detect loading of module ideapad-laptop.

Now it detects loading of this module only if running with

# kedr start ideapad_laptop

So KEDR user should be aware of special processing of '-' character in module 
name. That is inconvenient.


```

-----
Original issue reported on code.google.com by `tsy...@gmail.com` on 2 Dec 2011 at 7:10

**Comment by GoogleCodeExporter** [comment]
```
Fixed in the repository.
```

-----
Original comment by `euspec...@gmail.com` on 13 Aug 2012 at 2:36
* Changed title: **(Fixed in Hg) KEDR should detect loading of module with '-' in its name**
* Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
-----
Original comment by `euspec...@gmail.com` on 27 Sep 2012 at 2:08
* Changed title: **(Fixed in KEDR 0.4) KEDR should detect loading of module with '-' in its name**
* Changed state: **Fixed**
